test/unit/globecontrol.js

```diff
@@ -180,10 +180,10 @@ describe('GlobeControls', function () {
 
     it('mouse wheel', function () {
         const startRange = controls.getRange();
-        event.wheelDelta = -10;
+        event.deltaY = -10;
         controls.onMouseWheel(event);
         assert.ok(controls.getRange() > startRange);
-        event.wheelDelta = 10;
+        event.deltaY = 10;
         controls.onMouseWheel(event);
         controls.onMouseWheel(event);
         assert.ok(controls.getRange() < startRange);
```

Comment:
Could you please switch `<` for `>`, because of the same reasons mentioned in the previous comment ?